panic due to duplicate flag

```
$ ./etcdadm 
panic: init flag redefined: disk-priorities

goroutine 1 [running]:
github.com/spf13/pflag.(*FlagSet).AddFlag(0xc0001ae200, 0xc00025b5e0)
        /home/daniel/go/pkg/mod/github.com/spf13/pflag@v1.0.1/flag.go:836 +0x80c
github.com/spf13/pflag.(*FlagSet).VarPF(0xc0001ae200, 0xd871e0, 0xc000182d40, 0xc77f5c, 0xf, 0x0, 0x0, 0xc8064f, 0x1a, 0xc00025b540)
        /home/daniel/go/pkg/mod/github.com/spf13/pflag@v1.0.1/flag.go:819 +0x10b
github.com/spf13/pflag.(*FlagSet).VarP(...)
        /home/daniel/go/pkg/mod/github.com/spf13/pflag@v1.0.1/flag.go:825
github.com/spf13/pflag.(*FlagSet).StringArrayVar(0xc0001ae200, 0x13417e8, 0xc77f5c, 0xf, 0x132d1a0, 0x3, 0x3, 0xc8064f, 0x1a)
        /home/daniel/go/pkg/mod/github.com/spf13/pflag@v1.0.1/string_array.go:57 +0xce
sigs.k8s.io/etcdadm/cmd.init.3()
        /home/daniel/go/src/sigs.k8s.io/etcdadm/cmd/join.go:210 +0x2da
```

Looks like this is due to #121

/kind bug
/priority critical-urgent
/lifecycle active

--- a/cmd/join.go
+++ b/cmd/join.go
@@ -207,5 +207,5 @@ func init() {
 	joinCmd.PersistentFlags().StringVar(&etcdAdmConfig.ReleaseURL, "release-url", constants.DefaultReleaseURL, "URL used to download etcd")
 	joinCmd.PersistentFlags().StringVar(&etcdAdmConfig.CertificatesDir, "certs-dir", constants.DefaultCertificateDir, "certificates directory")
 	joinCmd.PersistentFlags().StringVar(&etcdAdmConfig.InstallDir, "install-dir", constants.DefaultInstallDir, "install directory")
-	initCmd.PersistentFlags().StringArrayVar(&etcdAdmConfig.EtcdDiskPriorities, "disk-priorities", constants.DefaultEtcdDiskPriorities, "Setting etcd disk priority")
+	joinCmd.PersistentFlags().StringArrayVar(&etcdAdmConfig.EtcdDiskPriorities, "disk-priorities", constants.DefaultEtcdDiskPriorities, "Setting etcd disk priority")
 }

